# This custom message defines a multi array of extended motor data.

std_msgs/MultiArrayLayout layout        	# Specification of data layout.
MotorDataExtended[] motor_data_extended		# Array of MotorDataExtended.
